spec/controllers/prompt_meme_controller_spec.rb

```diff
@@ -2,6 +2,42 @@
 
 describe Challenge::PromptMemeController do
   include LoginMacros
+  include RedirectExpectationHelper
+
+  describe "new" do
+    context "when the collection already has a challenge" do
+      before do
+        @collection = FactoryGirl.create(:collection, challenge: PromptMeme.new)
```

Comment:
We should use let to avoid sharing state between tests in instance variables. You don't need to update the existing instance variables though.